src/meltano/cli/utils.py

```diff
@@ -58,8 +58,7 @@ def print(self) -> None:
         if self.printed:
             return
 
-        logger.error(str(self), exc_info=self)
-        click.secho(str(self), fg="red", err=True)
+        logger.error(str(self), exc_info=self.__cause__)
```

Comment:
This skips the redundant stack frame for `CliError` and goes straight to the underlying error.